src/pydvl/utils/parallel/futures/ray.py

```diff
@@ -386,7 +386,7 @@ def flag_executor_shutting_down(self):
                 # We cancel the future's object references
                 # We cannot cancel a running future object.
                 for future in self.submitted_futures:
-                    ray.cancel(future.object_ref)
+                    ray.cancel(future.object_ref)  # type: ignore
```

Comment:
I couldn't find the type definition for ray...Future

Reply:
`Future` comes from `concurrent.futures` and `object_ref` is set [at runtime](https://github.com/appliedAI-Initiative/pyDVL/blob/54d9f40dd29412d0e1f0de589e759b3c5418c177/src/pydvl/utils/parallel/futures/ray.py#L238) so typing wouldn't work anyway.